mobly/test_runner.py

```diff
@@ -472,7 +421,7 @@ def run_test_class(self, test_cls_name, test_cases=None):
                 self.results += e.results
                 raise e
 
-    def run(self, test_class=None):
+    def run(self, test_classes):
```

Comment:
Because for the plural case, which happens when test_class is None, 'run' used to import test files.
Now that the import test files functions are removed, 'run' should take a list instead.